cmds/tui.c

```diff
@@ -771,14 +833,19 @@ static struct tui_graph *tui_graph_init(struct uftrace_opts *opts)
 		top->name = basename(graph->ug.sess->exename);
 		top->nr_calls = 1;
 		top->total_time.min = -1ULL;
+		top->self_time.min = -1ULL;
 
 		list_for_each_entry(node, &graph->ug.root.head, list) {
 			top->total_time.sum += node->total_time.sum;
 			top->child_time += node->total_time.sum;
+
 			update_graph_time_stat(&top->total_time, node->total_time.sum);
+			update_graph_time_stat(&top->self_time, node->self_time.sum);
```

Comment:
At this point, I'm not sure whether this is correct.  At least each node should have `nr_calls == 1`.  Could you please add `ASSERT` to check the condition before calling the function?